atoma-proxy/src/main.rs

```diff
@@ -161,6 +166,23 @@ async fn main() -> Result<()> {
         shutdown_sender.clone(),
     );
 
+    let daemon_tcp_listener = TcpListener::bind(&config.daemon.service_bind_address)
+        .await
+        .context("Failed to bind daemon TCP listener")?;
+
+    let daemon_app_state = DaemonState {
+        atoma_state: AtomaState::new_from_url(&config.state.database_url).await?,
+    };
+
+    let daemon_handle = spawn_with_shutdown(
```

Comment:
I would rename the service to something else than `daemon`, maybe proxy service ?

Reply:
done